src/app/shared/dockstore.model.ts

```diff
@@ -36,6 +36,7 @@ export class Dockstore {
   static ANVIL_IMPORT_URL = 'https://anvil.terra.bio/#import-tool/dockstore';
   static BD_CATALYST_SEVEN_BRIDGES_IMPORT_URL = 'https://sb.biodatacatalyst.nhlbi.nih.gov/integration/trs/import';
   static BD_CATALYST_TERRA_IMPORT_URL = 'https://terra.biodatacatalyst.nhlbi.nih.gov/#import-tool/dockstore';
+  static CAVATICA_IMPORT_URL = 'https://cavatica.sbgenomics.com/integration/trs/import';
```

Comment:
Confused why this is showing up. It should already be in develop because I cherry-picked from there. Maybe GitHub just shows it anyway.